data/json/items/melee.json

```diff
@@ -1467,7 +1467,7 @@
     "price_postapoc": 0,
     "material": "aluminum",
     "techniques": "WBLOCK_2",
-    "flags": "SHEATH_SWORD",
+    "flags": [ "SHEATH_SWORD", "FRAGILE_MELEE" ],
```

Comment:
There are two main things that make cheap swords bad.  One is using a steel that wont hold an edge, another is the tang of the blade not extending through the hilt.  If were assuming the are extremely cheap swords, the latter is 5he case and fragile melee is called for.

I'm fine with that, I just wanted to be specific about it.